### DISCUSSION TITLE
Convergence criteria

### DISCUSSION THREAD

**Discussion by jmeier** [idaholab/moose#29359] [Q&A Modules: Porous Flow] [answered]
Dear Moose community,

I'd like to ask a question on the very good explanations given here:
[mooseframework.inl.gov/modules/porous_flow/convergence.html](https://mooseframework.inl.gov/modules/porous_flow/convergence.html)

In the section on fluids the following equation is given:
$R=V \cdot \left|\kappa\right| \cdot \rho \cdot \epsilon / \mu$

Can you please clarify what $\left|\kappa\right|$ is? 

Cross-checking [here](https://mooseframework.inl.gov/source/auxkernels/PorousFlowDarcyVelocityComponentLowerDimensional.html) my guess is, that $\left|\kappa\right|$ is the determinant of the permeability tensor (so, a physical unit of [mass] · [length]^5 / [time] ?). For a gravel with a isotropic permeability of 1E-7 m² and default SI-units this would lead to $\left|\kappa\right| = 1E-21 m^6$. For water, a volume of interest of 1m³ and $\epsilon=1 Pa/m$ this would lead to $R=1E-15 kg \cdot m^5 / s$. 

But on the other hand, the value calculated above is much smaller than the $R_{fluid}=10E-9 \cdot V$ given in section "Scaling the variables" in [mooseframework.inl.gov/modules/porous_flow/convergence.html](https://mooseframework.inl.gov/modules/porous_flow/convergence.html).

Thanks for you help in advance.

Edit: Unit of R.

**Comment by GiudGiud** [comment]
@rpodgorney @1runer

**Comment by cpgr** [comment, marked as answer]
Pretty sure it's just an order of magnitude of the permeability, so just 1e-7 in your case.

**Reply by jmeier** [reply]
That is probably really the case. For gravel/sand/clay this gives:

For gravel ($\kappa \approx 1E-7 m²$):
$R = (1 m³) \cdot (1E-7 m²) \cdot (1000 kg/m³) \cdot (1 Pa/m) / (1 mPa \cdot s) = 1E-1 kg \cdot m / s $

For sand ($\kappa \approx 1E-12 m²$):
$R = (1 m³) \cdot (1E-12 m²) \cdot (1000 kg/m³) \cdot (1 Pa/m) / (1 mPa \cdot s) = 1E-6 kg \cdot m / s $

For clay ($\kappa \approx 1E-15 m²$):
$R = (1 m³) \cdot (1E-15 m²) \cdot (1000 kg/m³) \cdot (1 Pa/m) / (1 mPa \cdot s) = 1E-9 kg \cdot m / s $
